.semaphore/semaphore.yml

```diff
@@ -18,7 +18,7 @@ blocks:
         - name: Build and deploy image
           matrix:
             - env_var: GO_VERSION
-              values: ["1.19"]
+              values: ["1.19-alpine3.16", "1.22-alpine3.18"]
```

Comment:
I think what it does is to build and push both images.

The old one won't be actually needed soon enough but I figured a scenario to deal with.

From when we merge this until we'll apply it will take some time. I figured we could build both for a brief time in case we have Snyk complaining for something within the old image.

Having both within the Matrix eliminates this issue altogether. We would just need to rebuild the images.

This, ofc, will require a new PR (later) to actually remove `golang:1.19-alpine3.16`.

Reply:
Ah, but this won't work at all actually. Because we use `1.19` in our semaphore files. Yeah, you're right. Let me fix that.

Reply:
ended up forcing the new alpine image for both of them